api/src/main/java/marquez/db/mappers/SimpleLineageEdgeMapper.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package marquez.db.mappers;
+
+import static marquez.db.Columns.stringOrNull;
+import static marquez.db.Columns.stringOrThrow;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import lombok.NonNull;
+import marquez.common.models.DatasetId;
+import marquez.common.models.DatasetName;
+import marquez.common.models.JobId;
+import marquez.common.models.JobName;
+import marquez.common.models.NamespaceName;
+import marquez.db.LineageDao.SimpleLineageEdge;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+public final class SimpleLineageEdgeMapper implements RowMapper<SimpleLineageEdge> {
```

Comment:
some javadoc would be helpful

Reply:
Ah yes, this is a first draft. If it looks good enough I'll add doc. I also need to update the openapi spec

Reply:
added